sandbox/public/index.html

```diff
@@ -37,34 +37,34 @@
 
     <script>
       alloy("configure", {
-        orgId: 23784612874,
-        propertyId: 9999999,
+        orgID: 23784612874,
+        propertyID: 9999999,
         dataSet: "dataset1",
         shouldStoreCollectedData: 1,
         device: "Chrome-Mac",
-        collectionUrl: "https://edgegateway.azurewebsites.net",
+        // collectionUrl: "https://edgegateway.azurewebsites.net",
+        collectionUrl: "http://ex-edge.stable-stage.aam-npe.adobeinternal.net/v1",
         destinationsEnabled: true,
         debug: true
       });
 
       alloy("viewStart", {
         data: {
-          event: "page View",
-          pageName: "home"
+          pageName: ["home"]
```

Comment:
The gateway currently requires all data items to be arrays. In a discussion with Rares on Slack, it sounds like this will change soon.